lws_config.h.in

```diff
@@ -61,3 +61,5 @@
 
 /* use SHA1() not internal libwebsockets_SHA1 */
 #cmakedefine LWS_SHA1_USE_OPENSSL_NAME
+
+${LWS_SIZEOFPTR_CODE}
```

Comment:
ah ok I missed the `_CODE` part... I did not know that part. I have only ever seen people define it like `#cmakedefine LWS_SIZEOFPTR_CODE ${LWS_SIZEOFPTR_CODE}`